source/ProgArgs.cpp

```diff
@@ -496,6 +496,8 @@ void ProgArgs::defineAllowedArgs()
 			"Send downloaded objects directly to /dev/null instead of a memory buffer. This option "
 			"is incompatible with any buffer post-processing options like data verification or "
 			"GPU data transfer.")
+/*s3f*/	(ARG_S3IGNOREERRORS_LONG, bpo::bool_switch(&this->ignoreS3Errors),
```

Comment:
`/*s3f*/` needs to be changed to `/s3i*/`. as you were probably aware already, these comments are just for us humans to make it easier to find the right place so that the `--help-all` output is sorted alphabetically by option strings.